build_artifacts/v1/v1.0/v1.0.0/v1.0.0-beta/Dockerfile

```diff
@@ -80,4 +85,16 @@ RUN mkdir -p /var/run/supervisord && \
 USER $MAMBA_USER
 ENV PATH="/opt/conda/bin:/opt/conda/condabin:$PATH"
 WORKDIR "/home/${NB_USER}"
+
+# https://stackoverflow.com/questions/122327
+RUN SYSTEM_PYTHON_PATH=$(python3 -c "from __future__ import print_function;import sysconfig; print(sysconfig.get_paths().get('purelib'))") && \
+    # Move Spark and PySpark Kernel to conda env so that it appears in JupyterLab
+    #cp -a ${SYSTEM_PYTHON_PATH}/sparkmagic/kernels/sparkkernel /opt/conda/share/jupyter/kernels/ && \
```

Comment:
Can we remove the commented-out `cp` commands?

Reply:
Yup will do it in next commit 👍